### DISCUSSION TITLE
Having trouble running app locally

### DISCUSSION THREAD

**Discussion by eshdotcom** [UW-FLOM/flom-react#105] [Q&A] [answered]
Hi team, I am having some trouble running flom-react locally. Here are the steps I've taken so far:

- In the directory `flom-react/client`, I built a docker image using `docker build -t flom-react:latest .` 
- I ran the docker container using `docker run --name flom1 -d -p 3000:80 flom-react:latest`

Although I see the container in the list of running containers (`docker ps`), port 3000 does not display the application. Please let me know if I am missing a step in my setup.

**Comment by VulpesVulpes825** [comment]
Please read the readme.md before asking so we can save everyone’s time. Run `yarn dev:setup` in root project folder, not in client folder, to start development containers. For more commands, check package.json located in the root project folder.

**Reply by eshdotcom** [reply]
Over the weekend I attempted to run `yarn dev:setup` in the root project folder, but I received an error: `Usage Error: Couldn't find a script named "dev:setup".`

**Reply by VulpesVulpes825** [reply]
There might be an issue with your yarn or node version, as GitHub can run `yarn dev:setup` without any issue.
See https://github.com/UW-FLOM/flom-react/runs/4363051918?check_suite_focus=true

**Reply by VulpesVulpes825** [reply]
Are you sure you are on the project root folder, rather than in client? Because it works on my end, and via GitHub Action. What are your yarn and node version? Both yarn classic and yarn 3.0 do run without error. Maybe also check the semicolon?

**Reply by VulpesVulpes825** [reply]
You can also try `yarn dev:start`, which skip the local dependency installation and go straight to building docker image.

**Comment by VulpesVulpes825** [comment, marked as answer]
Issue resolved via Zoom meeting.
